conf/distro/include/fsl-base.inc

```diff
@@ -10,8 +10,15 @@ TARGET_VENDOR = "-fsl"
 
 DISTROOVERRIDES = "fsl"
 
-# Use NXP BSP for default
+# Use NXP BSP for default.
+# The goal is for machine configs to default to mainline BSP settings. The
+# settings here override any such defaults so the build is pure NXP BSP.
 IMX_DEFAULT_BSP = "nxp"
+UBOOT_MAKE_TARGET = "u-boot.imx"
+UBOOT_SUFFIX = "imx"
+SPL_BINARY_mx6 = ""
+SPL_BINARY_mx7 = ""
```

Comment:
I believe we'll need to use an indirect variable to provide a default and allow the board to override it.

That would be something like:

NXP_UBOOT_MAKE_TARGET = "u-boot.imx"
UBOOT_MAKE_TARGET ?= "${NXP_UBOOT_MAKE_TARGET}"

So if the machine overrides, it'll still use its set value but default to the distro one. Does it sound right?

Reply:
Actually, no this didn't work for me. I adjusted the lines directly in `fsl-base.inc` and ended up with the same problem I'm trying to fix, that the `all` setting in `imx6qdlsabresd.conf` is taken regardless of the setting here in the distro config.